package.json

```diff
@@ -12,16 +12,16 @@
     "packages/blog-example"
   ],
   "scripts": {
-    "lint": "yarn run lint:prettier && yarn run lint:eslint",
-    "lint:prettier": "node --max-old-space-size=4096 node_modules/.bin/prettier --check \"(packages|scripts)/**/*.{js,ts,tsx}\"",
-    "lint:eslint": "node --max-old-space-size=4096 node_modules/.bin/eslint --quiet --ext ts,tsx packages scripts",
-    "lint:fix": "node --max-old-space-size=4096 node_modules/.bin/prettier --write --check \"(packages|scripts)/**/*.{js,ts,tsx}\" && eslint --ext ts,tsx --fix packages scripts",
-    "typecheck": "yarn workspaces run tsc --noEmit",
-    "build": "yarn workspaces run build",
+    "lint": "pnpm lint:prettier && pnpm lint:eslint",
+    "lint:prettier": "NODE_OPTIONS=\"--max-old-space-size=4096\" prettier --check \"(packages|scripts)/**/*.{js,ts,tsx}\"",
```

Comment:
@thegedge I figured out a way to not have to pull out the right file from the node package binary by just using the `NODE_OPTIONS` env var to set the options, instead of running the node binary explicitly. Vibe?

Reply:
I do! I thought I tried that and it complained about not being able to put `--max-old-space-size` in NODE_OPTIONS. I'll try it again though.